static_src/stores/user_store.js

```diff
@@ -270,6 +285,27 @@ export class UserStore extends BaseStore {
         break;
       }
 
+      case userActionTypes.CURRENT_USER_ROLES_RECEIVED: {
```

Comment:
As stated above in the actions, let's remove this for now for the merge and see if we need if after.